storage/wdpost_run.go

```diff
@@ -31,6 +31,7 @@ import (
 	"github.com/filecoin-project/lotus/chain/types"
 )
 
+// failPost records a failure in the journal.
 func (s *WindowPoStScheduler) failPost(err error, ts *types.TipSet, deadline *dline.Info) {
```

Comment:
I find this name a bit confusing. `failPost` sounds like we are failing a PoSt or something like that, given that we are in the `WindowPoStScheduler`.

Wouldn't `recordFailure` or `publishFailure` be better, given that this seems to be an auxiliary method to just log and record a tract of what happened?

Reply:
Makes sense.